.github/workflows/integration-test-allowed-namespaces.yml

```diff
@@ -0,0 +1,50 @@
+name: Integration-Test-Allowed-Namespaces
```

Comment:
the `integration-tests*.yml` workflows seem to differ only on the applied values.yaml, and the invoked `make integration-test-...` command - would it be an option to keep one `integration-test.yml` file where the test to be executed is passed as a parameter?

Reply:
That's true, but merging it together adds additional complexity to the workflow, this way it looks simple and is easy to extend with additional parameters if needed in the future. One thing we can do:

Currently 3 workflows have the same parameters (2 have different). We can at least merge these 3. Would this work fro you?

Reply:
sure, that would be fine with me :)